source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -455,6 +456,9 @@ It is allowed to use ``directConnection=true`` in conjunction with the
 verify that setName matches the specified name, as per
 `verifying setName with TopologyType Single`_.
 
+When the ``replicaSet`` URI option is provided, the driver MUST throw an
```

Comment:
I'd prefer not to duplicate this here as it's already mentioned in the LB spec.

Reply:
Removed.